extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -174,7 +174,7 @@ export const StateProperty = {
 
 /** @const @enum {string} */
 export const Action = {
-  ADD_TO_ACTIONS_WHITELIST: 'addToActionsWhitelist',
+  ADD_TO_ACTIONS_ALLOWLIST: 'addToActionsAllowlist',
```

Comment:
Changes to this file need @ampproject/wg-stories approval. I can't verify if this enum string is leaked to external code, or if this is truly internal.

Reply:
Adding a @ampproject/wg-stories member to review.